### PR TITLE
Be get fixtures

### PR DESCRIPTION
**Pull request by Stephen0wen**
Completed  GET /api/fixtures including queries

### CODE REVIEW

File: back-end/controllers/fixtures.controller.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const { selectAllFixtures } = require("../models/fixtures.model");
+
+exports.getFixtures = (request, response, next) => {
+    selectAllFixtures(request.query)
+        .then((fixtures) => {
+            response.status(200).send({ fixtures });
+        })
+        .catch((error) => {
+            console.log(error);
```
Comment (HanzoDaBoss) [inline back-end/controllers/fixtures.controller.js]: Remove console log after you add in error handling in a future pr

File: back-end/models/fixtures.model.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        division = false;
+    }
+
+    if (division) {
+        sqlString += `
+      AND team1.team_division = $${insertposition++}`;
+        sqlArr.push(division);
+    }
+
+    sqlString += ";";
```
Comment (HanzoDaBoss) [inline back-end/models/fixtures.model.js]: You can remove this

File: back-end/models/fixtures.model.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        sqlArr.push(match_status);
+    }
+
+    if (team_id && insertposition === 1) {
+        sqlString += `
+    WHERE (team1_id = $${insertposition++}
+      OR team2_id = $${insertposition++})`;
+        sqlArr.push(team_id);
+        sqlArr.push(team_id);
+        team_id = false;
```
Comment (HanzoDaBoss) [inline back-end/models/fixtures.model.js]: I commented line 41 out and the tests seem to pass, maybe try combining the match status and team_id queries in the testing to see if it still works?

File: back-end/models/fixtures.model.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        OR team2_id = $${insertposition++})`;
+        sqlArr.push(team_id);
+        sqlArr.push(team_id);
+    }
+
+    if (division && insertposition === 1) {
+        sqlString += `
+    WHERE team1.team_division = $${insertposition++}`;
+        sqlArr.push(division);
+        division = false;
```
Comment (HanzoDaBoss) [inline back-end/models/fixtures.model.js]: Same as previous comment

**Review by HanzoDaBoss — COMMENTED** [review]
Well done stephen, left some comments which are very minor changes

File: back-end/models/fixtures.model.js (hunk 2 above)
Comment (Stephen0wen) [reply back-end/models/fixtures.model.js]: Will the query run if I remove it?

Comment (HanzoDaBoss) [reply back-end/models/fixtures.model.js]: Yeah the tests do work without line 65

Comment (HanzoDaBoss) [reply back-end/models/fixtures.model.js]: its a node postgres thing where if you forget to add a semi colon it does it for you i believe

Comment (Stephen0wen) [reply back-end/models/fixtures.model.js]: Oh that's cool

**Review by HanzoDaBoss — APPROVED** [review]

**Pull request merged**